command/operator_rekey.go

```diff
@@ -417,17 +458,29 @@ func (c *OperatorRekeyCommand) provide(client *api.Client, key string) int {
 		return 2
 	}
 
+	var started bool
+	var nonce string
+
+	switch status.(type) {
```

Comment:
Since `updateFn` and `statusFn` are now returning `interface{}`, it may be prudent to add a default case to this switch and the one on 546 to error out immediately if the response is not an expected type.

Reply:
I thought about that. We have pretty positive control over what's there, which is why I didn't add it, but I can.